etc/Brewfile

```diff
@@ -0,0 +1,11 @@
+# deps for our macos.yml CI job
+
+tap "viamrobotics/brews"
+
+brew "pkg-config"
+brew "viamrobotics/brews/nlopt-static"
+brew "x264"
+brew "jpeg-turbo"
+brew "ffmpeg"
+# tflite needs to be last (why?)
```

Comment:
Some of the libraries TFlite builds against (and there are MANY) might get updated by the build/inclusion of other things above. E.g. before pkg-config is pulled in, it might build with the system pkg-config, which would then deliver completely different versions (from the outer system) compared to what brew would deliver. Less of a danger on MacOS, but until/unless we explicitly require EVERYTHING for the tflite formula that it actually uses, it's "fragile"

Reply:
yup it's fragile

tried this approach with x264 or ffmpeg and it broke immediately. I think nlopt and tflite work for this because we consume the static builds, but will still break at some point